internal/chrome/chrome.go

```diff
@@ -76,26 +76,15 @@ func (s *CookieStore) ReadCookies(filters ...kooky.Filter) ([]*kooky.Cookie, err
 			return err
 		}
 
-		encrypted_value, err := row.BytesStringOrFallback(`encrypted_value`, nil)
```

Comment:
don't remove filter function. A rework of the filter order should be a separate PR and without API removal.

Reply:
He just encapsulated the function and did not remove this part of the logic. This part was modified by another author who forked this project, and I gave him the code. As a result, the code on both sides is unbalanced.